docs/source/getting_started/content/con_understanding-codeready-containers-configuration.adoc

```diff
@@ -0,0 +1,10 @@
+[id="understanding-codeready-containers-configuration_{context}"]
+= Understanding {prod} configuration
+
+Both the [command]`{bin}` binary and the {prod} virtual machine can be configured using the [command]`{bin} config` command.
+The [command]`{bin} config` command requires a subcommand to act on the configuration.
+The available subcommands are `get`, `set,` `unset`, and `view`.
```

Comment:
`get` and `unset` are not documented in the rest of this PR, dunno if that's intentional?

Reply:
Good point. I'll add documentation for those.

Reply:
I've added documentation for `crc config get <property>`, but I believe `unset` more or less explains itself.